order/views.py

```
@@ -6,7 +6,7 @@
 from django.db import transaction
 from .models import Order, OrderItem
 from product.models import Product
-from user.models import CartItem  # ✅ user/models.py의 CartItem 사용
+from user.models import CartItem, User  # ✅ user/models.py의 CartItem 사용
```

Ruff (0.12.2)

`user.models.User` imported but unused

Remove unused import: `user.models.User`

(F401)